core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.support;
+
+/**
+ *
+ * @author     Samrat.Dhillon
```

Comment:
Remove author tag

Reply:
@davsclaus pushed changes matching master. I see changes in master trigger a build. Is it possible to verify changes in maintenance branch.

Reply:
There CI just do a build and code format testing - no full test run. That is only for code merged.

Reply:
And this is the danger of doing changes in core in a LTS release - eg we prefer not to. But this fixes a real bug for distributed tracing that is hard to "notice" due thread concurrency issues etc.

Reply:
@davsclaus Then please dont merge it yet. Let me rethink an easier way for LTS.

Reply:
Closing this in lieu of PR#5435